resume_pj/css/mainR.css

```diff
@@ -15,21 +14,23 @@ a {
     ;
 }
 
-.flex-nav{
-    display: flex;
-    justify-content: space-between;
-    padding: 5vh;
+
+.hello-section {
+    margin-top: -100px;
+    padding-top: 120px;
+    background-color:  #E6DACD ;
 }
-h1 h2 h3 h4 h5 h6 {
+
+h1, h2, h3, h4, h5, h6 {
     font-family: "https://fonts.google.com/specimen/Poppins";
 }
```

Comment:
You can't put the link to a font directly in your CSS. This should be `font-family: 'Poppins', sans-serif;` and the link to t he font should be imported into your HTML file in the `head` - see instructions on the right panel here: https://fonts.google.com/specimen/Poppins?sidebar.open=true&selection.family=Poppins:wght@200